scripts/check.py

```diff
@@ -133,7 +182,7 @@ def _check_tool(cfg: config.Config, bin_name: str, name: str,
 def check_prerequisites(cfg: config.Config) -> None:
     """Check availability and versions of all required prerequisite software."""
     is_ok = True
-    is_ok = is_ok and _check_host_compiler(cfg)
+    is_ok = is_ok and _check_toolchain(cfg, cfg.host_toolchain)
```

Comment:
Since `host_toolchain` is a member of `cfg` anyway, consider taking `cfg` as the single argument in `_check_toolchain` and then retrieving `host_toolchain` inside it.

Reply:
When cross compiling we will need two different toolchains, so I would prefer not to change this.

Reply:
Ok, so LGTM